Lesson_03/l03_h01_peoples.py

```diff
@@ -0,0 +1,80 @@
+"""HW:3[Volkov Anton].
+
+You have list of tuples.
+Each tuple represents: name, age, some sum, last name, sex
```

Comment:
```suggestion
"""
HW:3[Volkov Anton].

You have list of tuples.
Each tuple represents: name, age, some sum, last name, sex
```

Reply:
This will solve such linter error 
./T-HOMETASKS/L03/l03_h01_AntonVolkov.py:1:1: D205 1 blank line required between summary line and description